au/math_test.cc

```diff
@@ -713,8 +713,8 @@ TEST(CeilIn, SupportsDifferentOutputTypes) {
 }
 
 TEST(InverseAs, HandlesIntegerRepCorrectly) {
-    constexpr auto period = inverse_as(milli(seconds), hertz(40));
-    EXPECT_THAT(period, SameTypeAndValue(milli(seconds)(25)));
+    constexpr auto period = inverse_as(micro(seconds), hertz(40));
```

Comment:
Comment: This PR mentions round trip data loss, but operations like `inverse_as(hertz, micro(seconds(25'000)))` don't compile.

Reply:
Misplaced `)`.  This compiles:

```cpp
inverse_as(hertz, micro(seconds)(25'000))
```

It hadn't occurred to me before that people might try applying a prefix to a quantity, not a unit, but I can see how this syntax error would be an easy one to make.  :thinking: